cypress/integration/pages/frontPage/tests.js

```diff
@@ -6,6 +6,32 @@ const serviceHasIndexAlsos = service => service === 'thai';
 // Limiting to one service for now
 const serviceHasPublishedPromo = service => service === 'persian';
 
+// Check for valid useful links
+const shouldDisplayUsefulLinks = pageData => {
+  let usefulLinksItems = [];
+  const contentTypes = [
+    'Text',
+    'Feature',
+    'Audio',
+    'Video',
+    'Gallery',
+    'Guide',
+  ];
+  const usefulLinks = pageData.find(data => {
+    return data.type === 'useful-links';
+  });
+
+  if (usefulLinks) {
+    usefulLinksItems = usefulLinks.items.filter(item => {
+      return (
+        item.assetTypeCode === 'PRO' && contentTypes.includes(item.contentType)
```

Comment:
Should content type not be 'Guide' for useful links? Or was there a decision to support all of the types from the `contentTypes` array above?

Reply:
Yes, for useful links the contentType should be `Guide`. Changed the function to only filter `item.contentType === 'Guide'`